core-infrastructure/src/db/Core.Database/Scripts/051-CreateTrustFinancialTable.sql

```diff
@@ -0,0 +1,139 @@
+            TotalEducationalSuppliesCosts                            decimal(16, 2)      NULL,
+            ExaminationFeesCosts                                     decimal(16, 2)      NULL,
+            LearningResourcesNonIctCosts                             decimal(16, 2)      NULL,
+            LearningResourcesIctCosts                                decimal(16, 2)      NULL,
+            TotalPremisesStaffServiceCosts                           decimal(16, 2)      NULL,
+            CleaningCaretakingCosts                                  decimal(16, 2)      NULL,
+            MaintenancePremisesCosts                                 decimal(16, 2)      NULL,
+            OtherOccupationCosts                                     decimal(16, 2)      NULL,
+            PremisesStaffCosts                                       decimal(16, 2)      NULL,
+            TotalUtilitiesCosts                                      decimal(16, 2)      NULL,
+            EnergyCosts                                              decimal(16, 2)      NULL,
+            WaterSewerageCosts                                       decimal(16, 2)      NULL,
+            AdministrativeSuppliesNonEducationalCosts                decimal(16, 2)      NULL,
+            TotalGrossCateringCosts                                  decimal(16, 2)      NULL,
+            TotalNetCateringCostsCosts                               decimal(16, 2)      NULL,
+            CateringStaffCosts                                       decimal(16, 2)      NULL,
+            CateringSuppliesCosts                                    decimal(16, 2)      NULL,
+            TotalOtherCosts                                          decimal(16, 2)      NULL,
+            DirectRevenueFinancingCosts                              decimal(16, 2)      NULL,
+            GroundsMaintenanceCosts                                  decimal(16, 2)      NULL,
+            IndirectEmployeeExpenses                                 decimal(16, 2)      NULL,
+            InterestChargesLoanBank                                  decimal(16, 2)      NULL,
+            OtherInsurancePremiumsCosts                              decimal(16, 2)      NULL,
+            PrivateFinanceInitiativeCharges                          decimal(16, 2)      NULL,
+            RentRatesCosts                                           decimal(16, 2)      NULL,
+            SpecialFacilitiesCosts                                   decimal(16, 2)      NULL,
+            StaffDevelopmentTrainingCosts                            decimal(16, 2)      NULL,
+            StaffRelatedInsuranceCosts                               decimal(16, 2)      NULL,
+            SupplyTeacherInsurableCosts                              decimal(16, 2)      NULL,
+            CommunityFocusedSchoolStaff                              decimal(16, 2)      NULL,
+            CommunityFocusedSchoolCosts                              decimal(16, 2)      NULL,
```

Comment:
`DirectRevenueFinancing`, `CommunityFocusedSchoolStaff`, `CommunityFocusedSchoolCosts` need removing; for maintained schools only (as per `cost_category_map` configuration).